File: templates/Uwp/_composition/MVVMLight/Project.SplitView.TabbedNavVB/ViewModels/ViewModelLocator_postaction.vb
Namespace ViewModels
    Public Class ViewModelLocator
        Private Sub New()
            '^^
            '{[{
            SimpleIoc.[Default].Register(Of ShellViewModel)()
            '}]}
        End Sub
        '{[{

        Public ReadOnly Property ShellViewModel As ShellViewModel
            Get
                Return SimpleIoc.[Default].GetInstance(Of ShellViewModel)()
            End Get
        End Property
        '}]}
    End Class
End Namespace

File: templates/Uwp/_comp/MVVMLight/Project.SplitView.TabbedNav/ViewModels/ViewModelLocator_postaction.cs
namespace Param_RootNamespace.ViewModels
{
    public class ViewModelLocator
    {
        private ViewModelLocator()
        {
//^^
//{[{
            SimpleIoc.Default.Register<ShellViewModel>();
//}]}
        }

//{[{
        public ShellViewModel ShellViewModel => SimpleIoc.Default.GetInstance<ShellViewModel>();
//}]}
    }
}
